### DISCUSSION TITLE
Metronome in Beatmap Editor

### DISCUSSION THREAD

**Discussion by nahkd123** [ppy/osu#15814] [Ideas]
I just looked at [Beatmap Editor design](https://www.figma.com/file/ytnnne2TH8Z956Jxhiypqq/Beatmap-Editor-2?node-id=0%3A1) and I saw no "Toggle Metronome" (or something like that). Adding metronome in beatmap editor helps a lot on aligning markers.

I would suggest putting toggle metronome checkbox on this list:
![image](https://user-images.githubusercontent.com/20136708/143550611-e4c3004d-c879-46fd-9f7c-8fc371f779b2.png)

The metronome sound can be simple sine wave beeps, or with the sounds when you hover on osu! cookie.

**Comment by peppy** [comment]
A metronome would be welcomed, but those toggles are for visual elements in the timeline view. It shouldn't be displayed there.
